econml/inference/_inference.py

```diff
@@ -240,8 +243,11 @@ def effect_inference(self, X, *, T0, T1):
         e_stderr = self._prediction_stderr(cross_product(X, T1 - T0))
         d_y = self._d_y[0] if self._d_y else 1
         # d_t=1 here since we measure the effect across all Ts
+        # A dummy treatment names is passed in
```

Comment:
Consider rewording, since it is not actually passed in

```suggestion
        # Omit treatment name argument to fall back to the default
```